spec/lib/iam_ssoe_oauth/session_manager_spec.rb

```diff
@@ -19,6 +19,7 @@
       before do
         VCR.use_cassette('iam_ssoe_oauth/introspect_active') do
           @user = session_manager.find_or_create_user
+          expect(@user.last_signed_in).not_to be_nil
```

Comment:
Could this go down into an `it` block? Might also want to check that it's a time in addition to not nil

Reply:
will do